tests/test_subscriber.py

```diff
@@ -221,23 +219,66 @@ def test_handle_notification_false(mock_messenger, mock_config, mock_create_inde
             "device_data": "device-data"
         }
     }
-    conf.dojot_persist_notifications_only = False
-    start_dojot_messenger(mock_config, p)
 
+    # test persist all boolean valued events
+    start_dojot_messenger(mock_config, p, False)
+
+    mock_create_notifications = mock_create_channel("dojot.notifications", "r")
```

Comment:
Why are you calling the mocks? I doesn't make sense for me!
You should check if the mocks are called by the method you are testing!